docs/guides/data-and-dbt-tests-report-ui.mdx

```diff
@@ -63,11 +66,13 @@ The rows are expandable, and on expansion show information about the last runs o
 - Affected rows
 - Time
 
-<img src="https://res.cloudinary.com/mintlify/image/upload/v1659304885/elementary/Test-Runs-video-demo_ubxssd.gif" alt="Test runs demo" />
+<img
+  src="https://res.cloudinary.com/mintlify/image/upload/v1659304885/elementary/Test-Runs-video-demo_ubxssd.gif"
+  alt="Test runs demo"
+/>
 
-## Configure Slack report
+## Exporting the report
```

Comment:
I would name it "Sharing the report", as this is the user's perspective. 
Also, this is what we want to incentivize them to do.

Reply:
Totally agree!